Dockerfile

```
@@ -17,6 +17,19 @@ WORKDIR /app
 FROM node:18-slim AS node_builder
 WORKDIR /renderer
 
+# Install required fonts and dependencies for canvas
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)